radiance_folder/model/scene/README.md

```diff
@@ -0,0 +1,50 @@
+# static scene
+
+`model/scene`
+
+This folder includes all the *static* geometries in the model which are not *apertures*.
+By default, the files must follow the certain naming convention as below:
+
+1. `*.rad`: This file only includes the geometries. It should not include any of the
+   modifiers. Modifiers should be included in `*.mat` file.
+2. `*.mat`: This file includes all the modifiers for geometries included in `*.rad`.
+3. `*.blk`: This file includes modifiers for geometries in `*.rad` that will
+   be used for direct sunlight calculation. For opaque modifiers the materials in `*.blk`
+   is usually a black plastic. For transparent and translucent modifiers the modifier
+   usually stays the same. Otherwise the light will be blocked in direct studies.
+
+You can change the naming convention for files by editing the config file in the model
+folder.
+
+All the geometries can be included in the same `.rad`, `.mat` and `.blk` files together
+but for this example we are separating them into separate files to make it easier to
+study the files.
+
+## NOTE
+
+Be careful not to include the apertures in this folder. The apertures will be treated
+differently in studies with aperture groups and any aperture in this folder can cause in
+double counting.
+
+
+In this sample case the room geometry as well as ground plane and the neighbor building
+are opaque. There is also the bottom part of the partition inside the room which is
+opaque.
```

Comment:
Why does it matter than they are opaque. Isn't the reason why they are in this folder because they are static and they aren't an aperture?